Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -437,10 +437,13 @@ GENERATE_SINGLE_SWIFT_INTEROP_FILE_YES = GENERATE_SINGLE_SWIFT_INTEROP_FILE;
 // FIXME: (rdar://149474443) Swift/Cxx interop does not respect CLANG_CXX_LANGUAGE_STANDARD = "c++2b";
 // FIXME: (rdar://103177280) "-no-verify-emitted-module-interface;" is needed to workaround a SwiftVerifyEmittedModuleInterface bug
 OTHER_SWIFT_FLAGS = $(inherited) $(OTHER_SWIFT_FLAGS_SUPPORTS_SWIFT_OBJCXX_INTEROP_$(WK_SUPPORTS_SWIFT_OBJCXX_INTEROP)) $(OTHER_SWIFT_FLAGS_MARK_FOUNDATION_AS_OBJC_$(WK_GENERATE_SINGLE_SWIFT_INTEROP_FILE);
-OTHER_SWIFT_FLAGS_SUPPORTS_SWIFT_OBJCXX_INTEROP_YES = -Xcc -std=c++2b -no-verify-emitted-module-interface;
+OTHER_SWIFT_FLAGS_SUPPORTS_SWIFT_OBJCXX_INTEROP_YES = -Xcc -std=c++2b -no-verify-emitted-module-interface $(OTHER_SWIFT_FLAGS_MINACCESS_INTERNAL);
 OTHER_SWIFT_FLAGS_SUPPORTS_SWIFT_OBJCXX_INTEROP_NO = ;
 OTHER_SWIFT_FLAGS_MARK_FOUNDATION_AS_OBJC_YES = -enable-experimental-feature RequiresObjC=Foundation;
 OTHER_SWIFT_FLAGS_MARK_FOUNDATION_AS_OBJC_NO = ;
+OTHER_SWIFT_FLAGS_MINACCESS_INTERNAL = ;
+OTHER_SWIFT_FLAGS_MINACCESS_INTERNAL[sdk=macosx26.4*] = -Xfrontend -emit-clang-header-min-access -Xfrontend internal;
```

Comment:
@emw-apple hi! What's the best way to pass this option for _all_ platforms for 26.4+? I feel like I'm missing something extremely obvious!